vendor/gotest.tools/v3/assert/assert.go

```diff
@@ -179,6 +175,15 @@ func logFailureFromBool(t TestingT, msgAndArgs ...interface{}) {
 	t.Log(format.WithCustomMessage(failureMessage+msg, msgAndArgs...))
 }
 
+func failureMsgFromError(err error) string {
```

Comment:
How come this change is in with the vendor bump? The code does not look equivalent.

Reply:
What do you mean? Looks to be in the diff; https://github.com/gotestyourself/gotest.tools/compare/v3.0.1...v3.0.2

Reply:
Oh sorry, I thought this was moby code when I looked at this...
More ☕️

Reply:
Guess I'm not used to seeing vendor code expanded :)

Reply:
Oh! Yes, I noticed that as well, seems as if GitHub's "hiding vendor code" may not understand non-github.com vendor paths 🤔